src/send_data.py

```diff
@@ -1,71 +1,43 @@
-from sensor_temp import getTemp
-from sensor_motion import getMotion
-from alarm import alarm_on
-from button import button_pressed
 import requests
-from pprint import pprint
 from requests import RequestException
-import time
-import datetime
-import os
-from dotenv import load_dotenv
-
-load_dotenv()
-
-URL = os.getenv("SERVER_URL")
-DEVICE_ID = '1'
+from pprint import pprint
+from time import sleep
+# import os
+# from dotenv import load_dotenv
 
-## SEND DATA OUT
-def create_sensor_data_dict():
-    """Create data from multiple sensors
+# load_dotenv()
 
-    Raises:
-        e: Failed to get data from sensor
+# URL = os.getenv("SERVER_URL")
+# DEVICE_ID = '1'
 
-    Returns:
-        dict: Dictionary of sensor data
-    """
-    
+def send_data(data, URL, DEVICE_ID):
     try:
-        sensor_temp = getTemp()
-        temp_c = round(sensor_temp["temperature_c"], 1)
-        temp_f = round(sensor_temp["temperature_f"], 1)
-        humidity = sensor_temp["humidity"]
-        motion = getMotion()
-        alarm = False #alarm_on()
-        button = button_pressed()
-        timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+        print('Sending')
+        # pprint(data)
+        res = requests.post(f'{URL}/device-data/{DEVICE_ID}', json = data)
+        print(res.text)
 
-        return [{
-            'temperature_c': temp_c,
-            'temperature_f': temp_f,
-            'humidity': humidity,
-            'motion': motion,
-            'alarm': alarm,
-            'button': button,
-            'created_at': timestamp
-        }]
+        try:
+            res.raise_for_status()
+            
+        except RequestException as e:
+            print("Request failed: ", e)
+            pass
+
+        print('Successfully send the data to server')
 
     except Exception as e:
-        raise e
+        print("Some failure occurred when monitoring sensor")
```

Comment:
Print `e` too.
When we faced this `print` we cannot know exactly where did the error come from.

Reply:
I think this was something you added.   Would you like us to delete the print?

Reply:
Done